src/Schema/Exception/NotEnoughValidSchemas.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace League\OpenAPIValidation\Schema\Exception;
+
+use Throwable;
+
+// Indicates that data was not matched against a combination of schema keywords
+class NotEnoughValidSchemas extends KeywordMismatch
+{
+    /** @var Throwable[] */
+    protected $innerExceptions = [];
+
+    /**
+     * @param mixed       $data
+     * @param Throwable[] $innerExceptions
+     *
+     * @return self
+     */
+    public static function fromKeywordWithInnerExceptions(
+        string $keyword,
+        $data,
+        array $innerExceptions,
+        ?string $message = null,
+        ?Throwable $prev = null
```

Comment:
should we care about `prev` here? make no sense for me

Reply:
you're right; removed it.